packages/manager/modules/netapp/src/dashboard/volumes/dashboard/snapshots/restore/controller.js

```diff
@@ -3,19 +3,28 @@ import { SNAPSHOT_STATUS } from './constants';
 
 export default class NetAppVolumesDashboardSnapshotsRestoreController {
   /* @ngInject */
-  constructor($translate, NetAppRestoreVolumeService) {
+  constructor($translate, NetAppRestoreVolumeService, coreConfig) {
     this.$translate = $translate;
+    this.user = coreConfig.getUser();
     this.NetAppRestoreVolumeService = NetAppRestoreVolumeService;
   }
 
   $onInit() {
+    const language = this.user.language.replace('_', '-');
```

Comment:
> Language
Allowed: cs_CZ┃de_DE┃en_AU┃en_CA┃en_GB┃en_IE┃en_US┃es_ES┃fi_FI┃fr_CA┃fr_FR┃fr_MA┃fr_SN┃fr_TN┃it_IT┃lt_LT┃nl_NL┃pl_PL┃pt_PT

Can we consider the locale on Manager instead?